include/nccl_ofi_freelist.h

```diff
@@ -196,7 +196,7 @@ int nccl_ofi_freelist_add(nccl_ofi_freelist_t *freelist,
 /*
```

Comment:
My comment in the discussion of https://github.com/aws/aws-ofi-nccl/pull/521/commits/05a7cd751dd93ccbbc7133a888880f97bff13452 remains.

Reply:
I've made that change, let me know if you like this variant better.